tests/tests/tier0/bluechi-change-port-with-p-cmd-option/test_bluechi_change_port_with_p_cmd_option.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+import os
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.machine import BluechiControllerMachine, BluechiAgentMachine
+from bluechi_test.config import BluechiControllerConfig, BluechiAgentConfig
+from bluechi_test.systemctl import SystemCtl
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    bluechi_agent_str = "bluechi-agent"
+    bluechi_controller_str = "bluechi-controller"
+
+    # change controller port to 8421
+    ctrl.exec_run(
+        f"sed -i 's|ExecStart=/usr/libexec/{bluechi_controller_str}|ExecStart=/usr/libexec/{bluechi_controller_str} -p 8421|' /usr/lib/systemd/system/{bluechi_controller_str}.service"
+    )
```

Comment:
This should be encapsulated in a dedicated method like the [restart_with_config_file](https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/machine.py#L127-L136).

Maybe it would be good to provide a `change_file(sed_cmd: str, file_location: str)` - but that is not necessary and can wait.

Reply:
waiting for https://github.com/eclipse-bluechi/bluechi/pull/828 merged, seems better to use new `Services` class to do this